### PR TITLE
Add TA-88047

### PR DESCRIPTION
**Pull request by rmloveland**

### CODE REVIEW

**Review by rytaft — APPROVED** [review]
:lgtm: Thanks!

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15178#-:-NCbDC0oEFMba401n56j:b-g2a5fb)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @msirek)

**Review by msirek — APPROVED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15178#-:-NCbYv2z-_mROsamRavf:bg7igkv)* status: :shipit: complete! 2 of 0 LGTMs obtained (waiting on @rmloveland)

File: advisories/a88407.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Technical Advisory 88047
+advisory: A-88047
+summary: Querying a <code>REGIONAL BY ROW</code> or partitioned multi-region table could produce incorrect results if the query had a <code>LIMIT</code> of less than 100,000 and used the inverted index.
```
Comment (ianjevans) [inline advisories/a88407.md:4]:
```suggestion
summary: Querying a <code>REGIONAL BY ROW</code> or partitioned multi-region table could produce incorrect results if the query has a <code>LIMIT</code> of less than 100,000 and uses an inverted index.
```

File: advisories/a88407.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+affected_versions: v22.1.0 to v22.1.7
+advisory_date: 2022-09-29
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.7](../releases/v22.1.html#v22-1-7), querying a [`REGIONAL BY ROW`](../stable/multiregion-overview.html#regional-by-row-tables) or [partitioned](../stable/partitioning.html) multi-region table with an [inverted index](../stable/inverted-indexes.html) could produce incorrect results if the query had a [`LIMIT`](../stable/limit-offset.html) of less than 100,000 and used the inverted index. In this case, it was possible that the [optimizer](../stable/cost-based-optimizer.html) would erroneously discard the predicate over the inverted index and therefore return some rows that did not match the predicate.
```
Comment (ianjevans) [inline advisories/a88407.md:15]:
```suggestion
In CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.7](../releases/v22.1.html#v22-1-7), querying a [`REGIONAL BY ROW`](../stable/multiregion-overview.html#regional-by-row-tables) or [partitioned](../stable/partitioning.html) multi-region table with an [inverted index](../stable/inverted-indexes.html) could produce incorrect results if the query has a [`LIMIT`](../stable/limit-offset.html) of less than 100,000 and uses an inverted index. In this case, it is possible that the [optimizer](../stable/cost-based-optimizer.html) will erroneously discard the predicate over the inverted index and therefore return some rows that do not match the predicate.
```

**Review by ianjevans — APPROVED** [review]
Just some suggestions to make it the descriptions present tense.

**Comment by nickvigilante** [comment]
@rmloveland Can you amend your commit and force-push? Retrying the build in Netlify did not properly sync the status to GitHub.

File: advisories/a88407.md (hunk 1 above)
Comment (rmloveland) [reply advisories/a88407.md:4]: Fixed!

File: advisories/a88407.md (hunk 2 above)
Comment (rmloveland) [reply advisories/a88407.md:15]: Fixed!

**Pull request merged**